app/assets/stylesheets/common/_markus.scss

```diff
@@ -948,24 +950,25 @@ nav {
 }
 
 .annotation_holder {
-  background: $orange;
-  border: 1px solid $black;
+  background: $annotation-holder;
+  border: 1px solid $sharp-line;
   opacity: 0.2;
   position: absolute;
   z-index: 1;
 }
 
 .annotation-holder-active {
-  background: $orange-transparent;
-  border: 3px solid $red;
+  background: $annotation-holder;
+  border: 3px solid $severe-error;
+  opacity: 0.2;

```

Comment:
It's okay to use the same colour as `.annotation_holder`, but let's increase the opacity a little to make this stand out.